src/IBM.WatsonDeveloperCloud.VisualRecognition.v3/VisualRecognitionService.cs

```diff
@@ -30,7 +30,7 @@ namespace IBM.WatsonDeveloperCloud.VisualRecognition.v3
 {
     public partial class VisualRecognitionService : WatsonService, IVisualRecognitionService
     {
-        const string SERVICE_NAME = "watson_vision_combined";
+        const string SERVICE_NAME = "visual_recognition";
```

Comment:
Same as with assistant. The service name is not changing in the IBM cloud

Reply:
These should both be fixed once https://github.com/watson-developer-cloud/dotnet-standard-sdk/pull/336 is merged.